### PR TITLE
Ops 1565/ux review

### PR DESCRIPTION
**Pull request by fpigeonjr**
## What changed

- style: ♻️ shows $ for amounts and if zero then no decimal place
- fix: 🐛 fixes bug where icon was bleeding thru alert

## Issue

#1565 

## How to test

1. e2e tests pass

## Screenshot

image

## Definition of Done Checklist
- [-] OESA: Code refactored for clarity
- [-] OESA: Dependency rules followed
- [X] Automated unit tests updated and passed
- [X] Automated integration tests updated and passed
- [X] Automated quality tests updated and passed
- [X] Automated load tests updated and passed
- [X] Automated a11y tests updated and passed
- [X] Automated security tests updated and passed
- [X] 90%+ Code coverage achieved
- [-] Form validations updated

### CODE REVIEW

**Review by jonnalley — APPROVED** [review]

**Review by johndeange — APPROVED** [review]

File: frontend/cypress/e2e/agreementList.cy.js [hunk 1]
```diff
@@ -16,7 +16,9 @@ it("loads", () => {
 });
 
 it("navigates to the approveAgreements page when the approve button is clicked", () => {
+    cy.wait(1000);
```
Comment (tdonaworth) [inline frontend/cypress/e2e/agreementList.cy.js:19]: Just curious, can we reduce this any? Having it at all is annoying, but would like to see if we can find a minimum amount required (500ms?). No biggie if not.

**Review by stevtek — APPROVED** [review]

File: frontend/src/components/Agreements/AgreementDetailsCards/AgreementTotalCard/AgreementTotalCard.jsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ const AgreementTotalCard = ({ total, subtotal, fees, procurementShop }) => {
                     value={subtotal}
                     displayType={"text"}
                     thousandSeparator={true}
-                    decimalScale={2}
+                    decimalScale={subtotal === 0 ? 0 : 2}
```
Comment (tdonaworth) [inline frontend/src/components/Agreements/AgreementDetailsCards/AgreementTotalCard/AgreementTotalCard.jsx:31]: I see this being used quite a bit, it's fairly simple, but curious if we want to wrap it as a formatting constant or something 🤔

Comment (fpigeonjr) [reply frontend/src/components/Agreements/AgreementDetailsCards/AgreementTotalCard/AgreementTotalCard.jsx:31]: good callout, probably a good candidate for our constants file

Comment (tdonaworth) [reply frontend/src/components/Agreements/AgreementDetailsCards/AgreementTotalCard/AgreementTotalCard.jsx:31]: maybe something like: 

```javascript
function getDecimalScale(value) {
  return value === 0 ? 0 : 2;
}

// Usage:
<decimalScale={getDecimalScale(data.subtotal)} />
```

File: frontend/cypress/e2e/agreementList.cy.js (hunk 1 above)
Comment (fpigeonjr) [reply frontend/cypress/e2e/agreementList.cy.js]: removed it and it still works

**Pull request merged**